server/db/mysql/adapter.go

```diff
@@ -719,8 +719,8 @@ func (a *adapter) AuthUpdRecord(uid t.Uid, scheme, unique string, authLvl auth.L
 		exp = &expires
 	}
 
-	_, err := a.db.Exec("UPDATE auth SET uname=?,authLvl=?,secret=?,expires=? WHERE uname=?",
-		unique, authLvl, secret, exp, unique)
+	_, err := a.db.Exec("UPDATE auth SET uname=?,authLvl=?,secret=?,expires=? WHERE userid=? AND scheme=?",
```

Comment:
It looks like mongo and rethinkdb still have the same problem.
Would it be possible for you to update those as well?

Reply:
it's weird mongodb and rethinkdb did not have this problem.